auth/src/main/java/com/firebase/ui/auth/ui/account_link/WelcomeBackPasswordPrompt.java

```diff
@@ -83,8 +82,6 @@ protected void onCreate(Bundle savedInstanceState) {
         TextView bodyTextView = ((TextView) findViewById(R.id.welcome_back_password_body));
         bodyTextView.setText(spannableStringBuilder);
 
-        ((TextInputLayout) findViewById(R.id.password_layout)).setPasswordVisibilityToggleEnabled(false);
```

Comment:
BTW, I've removed these because as of support lib 25.0.1, `false` is the default